### PR TITLE
Add User Avatar

### PR DESCRIPTION
**Pull request by bogadeji**
User Avatar component created

### CODE REVIEW

**Comment by chuksdozie** [comment]
no picture of the component you created

File: src/components/atoms/UserAvatar.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Box from "@mui/material/Box";
+
+function UserAvatar({
```
Comment (chuksdozie) [inline src/components/atoms/UserAvatar.js:5]: where is the picture of the component

**Review by chuksdozie — CHANGES_REQUESTED** [review]

File: src/components/atoms/UserAvatar.js (hunk 1 above)
Comment (bogadeji) [reply src/components/atoms/UserAvatar.js:5]: UserAvatar

Comment (bogadeji) [reply src/components/atoms/UserAvatar.js:5]: I used a random interet image

**Pull request closed without merging**